app/src/main/java/com/beemdevelopment/aegis/ui/views/AuditLogHolder.java

```diff
@@ -62,7 +62,7 @@ public void setData(AuditLogEntryModel auditLogEntryModel) {
             VaultEntry referencedVaultEntry = auditLogEntryModel.getReferencedVaultEntry();
             _auditLogEntryReference.setText(String.format("%s (%s)", referencedVaultEntry.getIssuer(), referencedVaultEntry.getName()));
         } else {
-            _auditLogEntryReference.setVisibility(View.GONE);
+            _auditLogEntryReference.setText(R.string.audit_log_entry_deleted);
```

Comment:
After fixing my previous comment I expect this line to be visible for all events that don't have a referenced vault entry, which is not what we want.